ocis/pkg/command/trash.go

```diff
@@ -0,0 +1,67 @@
+package command
+
+import (
+	"fmt"
+	"github.com/owncloud/ocis/v2/ocis/pkg/trash"
+
+	"github.com/owncloud/ocis/v2/ocis-pkg/config"
+	"github.com/owncloud/ocis/v2/ocis-pkg/config/configlog"
+	"github.com/owncloud/ocis/v2/ocis-pkg/config/parser"
+	"github.com/owncloud/ocis/v2/ocis/pkg/register"
+	"github.com/urfave/cli/v2"
+)
+
+func TrashCommand(cfg *config.Config) *cli.Command {
+	return &cli.Command{
+		Name:  "trash",
+		Usage: "ocis trash functionality",
+		Subcommands: []*cli.Command{
+			TrashPurgeOrphanedDirsCommand(cfg),
+		},
+		Before: func(c *cli.Context) error {
+			return configlog.ReturnError(parser.ParseConfig(cfg, true))
+		},
+		Action: func(_ *cli.Context) error {
+			fmt.Println("Read the docs")
+			return nil
+		},
+	}
+}
+
+func TrashPurgeOrphanedDirsCommand(cfg *config.Config) *cli.Command {
+	return &cli.Command{
+		Name:  "purge-orphaned-dirs",
```

Comment:
```suggestion
		Name:  "purge-empty-dirs",
```
They are not orphaned, they are empty. That is a difference. Please bulk change.